robotino_bringup/robots/tuw-robotino2/launch/tuw-robotino2.launch

```diff
@@ -15,96 +17,114 @@
 
-  <group ns="hand_controller">
-    <remap from="robot_description" to="/robot_description" />
-    <include file="$(find robotino_bringup)/launch/kclhand.launch" >
+    <!-- Simple node to move the kinect in the default position on robot's startup
+         It also exposes a service /tilt_controller/resetPosition -->
+    <node name="robotino_tilt_controller" pkg="robotino_driver" type="robotino_tilt_controller" output="screen">
+      <param name="tilt_command_topic" value="/tilt_controller/command" />
+      <param name="tilt_status_topic" value="/tilt_controller/state" />
+      <param name="startup_angle" value="0.7" />
+    </node>
+
+    <node name="robotino_pan_controller" pkg="robotino_driver" type="robotino_pan_controller" output="screen">
+      <param name="pan_command_topic" value="/pan_controller/command" />
+      <param name="pan_status_topic" value="/pan_controller/state" />
+      <param name="startup_angle" value="0.0" />
+    </node>
+
+    <group ns="real/robotino">
+	    <node name="robot_controller_node" pkg="uibk_robot_driver" type="robot_controller_node" output="screen">
+	    	  <remap from="cmd_rotatory" to="/cmd_rotatory"/>
+	    </node>
+      <node name="joint_states_relay" pkg="topic_tools" type="relay" args="joint_control/get_state /joint_states" cwd="node" respawn="true" output="screen"/>  
+    </group>
+
+    <group ns="hand_controller">
+      <remap from="robot_description" to="/robot_description" />
+      <include file="$(find robotino_bringup)/launch/kclhand.launch" >
+        <arg name="robot" value="$(arg robot)" />
+        <arg name="port" value="/dev/ttyHand"/>
+      </include>
+      <node name="joint_states_relay" pkg="topic_tools" type="relay" args="joint_states /joint_states" cwd="node" respawn="true" output="screen"/>
+    </group>
+
+    <node name="sensing" pkg="squirrel_sensing_node" type="sensing" args="/dev/ttyFingertips em1" output="screen"/>
+
+    <!--node name="wrist_safety" pkg="wrist_safety" type="wrist_safety" output="screen" /-->
+
+    <!--node name="squirrel_view_controller" pkg="robotino_driver" type="squirrel_view_controller" output="screen"/-->
+
+    <include file="$(find robotino_bringup)/launch/twist_mux.launch" >
       <arg name="robot" value="$(arg robot)" />
-      <arg name="port" value="/dev/ttyHand"/>
     </include>
-    <node name="joint_states_relay" pkg="topic_tools" type="relay" args="joint_states /joint_states" cwd="node" respawn="true" output="screen"/>
-  </group>
 
-  <node name="sensing" pkg="squirrel_sensing_node" type="sensing" args="/dev/ttyFingertips em1" output="screen"/>
+	</group>
 
-  <!--node name="wrist_safety" pkg="wrist_safety" type="wrist_safety" output="screen" /-->
+	<group>
+	  <machine name="scrat" address="$(arg hostname_scrat)" env-loader="$(arg env-script)" default="true" timeout="30"/>
 
-  <!--node name="squirrel_view_controller" pkg="robotino_driver" type="squirrel_view_controller" output="screen"/-->
+    <!-- kinect -->
```

Comment:
Ok, now I understand. 
Yes, the pc is not mounted yet and the kinect is still on robotino.